src/__tests__/inspect.test.ts

```diff
@@ -166,16 +165,37 @@ describe("inspect", () => {
 
   it("Use class names for the shortform of an object", () => {
     class Foo {
-      foo: string;
+      public foo: string;
 
       constructor() {
         this.foo = "bar";
       }
     }
 
-    expect(inspect([[[new Foo()]]])).toEqual("[[[[Foo]]]]");
+    expect(inspect([[new Foo()]])).toEqual("[[[Foo]]]");
 
     (Foo.prototype as any)[Symbol.toStringTag] = "Bar";
-    expect(inspect([[[new Foo()]]])).toEqual("[[[[Bar]]]]");
+    expect(inspect([[new Foo()]])).toEqual("[[[Bar]]]");
+  });
+});
+
+describe("createInspect", () => {
+  test("can increase the depth", () => {
```

Comment:
This changes were required to allow having deeper inspection on the tests.

Also made the defaults match `graphql-js`.